StripePayments/StripePaymentsObjcTestUtils/Resources/Mock Files/ElementsSession.json

```diff
@@ -195,5 +195,5 @@
   "shipping_address_settings": {
     "autocomplete_allowed": true
   },
-  "unactivated_payment_method_types": ["cashapp"]
+  "unactivated_payment_method_types": []
```

Comment:
TBD if this broke a test somewhere, but I didn't see why this json should have this particular unactivated payment method type and it was breaking a test.  If it's important, I can modify the stubbing code to inject the list of unactivated pm types.